### PR TITLE
Mental wellness

### PR DESCRIPTION
**Pull request by mimif10**
### Checklist

- [ ] Coding style [Trailing whitespaces in files & no unneeded whitespaces] :sparkles:
- [ ] Are there any .idea/ files or .gradle/ files in this PR? :notebook:

### What issue(s) does this PR close?

Closes

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -34,21 +35,15 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
-
-    buildFeatures {
-        viewBinding true
-    }
-   namespace 'com.example.life_is_well'
```
Comment (jacobburgo) [inline app/build.gradle]: We wanna make sure this is still in the project.

File: app/build.gradle [hunk 2]
```diff
@@ -34,21 +35,15 @@ android {
...
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-    implementation 'androidx.preference:preference:1.1.+'
-    implementation 'androidx.navigation:navigation-ui-ktx:2.5.2'
+    implementation 'androidx.fragment:fragment-ktx:1.5.2'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-}
+}
```
Comment (jacobburgo) [inline app/build.gradle]: We want this for deps:
```
dependencies {
    implementation 'androidx.core:core-ktx:1.8.0'
    implementation 'androidx.appcompat:appcompat:1.5.0'
    implementation 'com.google.android.material:material:1.6.1'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
    implementation 'androidx.preference:preference:1.1.+'
    implementation 'androidx.navigation:navigation-ui-ktx:2.5.2'
    implementation 'androidx.preference:preference-ktx:1.2.0'
    testImplementation 'junit:junit:4.13.2'
    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
}

```

Comment (jacobburgo) [reply app/build.gradle]: If you need something updated, that is fine, so long as the project is still working.

File: app/src/main/java/com/example/life_is_well/SettingsPageMain.kt [hunk 3]
```diff
@@ -20,7 +20,7 @@ class SettingsPageMain : AppCompatActivity() {
 
     class SettingsFragment : PreferenceFragmentCompat() {
         override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
-            setPreferencesFromResource(R.xml.root_preferences, rootKey)
+            43ewsazxzsetPreferencesFromResource(R.xml.root_preferences, rootKey)
```
Comment (jacobburgo) [inline app/src/main/java/com/example/life_is_well/SettingsPageMain.kt]:
```suggestion
            setPreferencesFromResource(R.xml.root_preferences, rootKey)
```

**Review by jacobburgo — CHANGES_REQUESTED** [review]
Just a couple things I wanna make sure about before merge.

@mimif10 You should be able to push up these changes from the MentalWellness Branch. If you have any questions, ping me on discord.

File: app/build.gradle (hunk 2 above)
Comment (jacobburgo) [reply app/build.gradle]: Disregard this. Recent merge resolved it.

File: app/build.gradle [hunk 4]
```diff
@@ -52,4 +48,4 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
```
[the remark this answers is not in the document]
Comment (jacobburgo) [reply app/build.gradle:48]: Sorry, looks like you don't have this one yet, but it should be in there.

File: app/build.gradle (hunk 1 above)
Comment (mimif10) [reply app/build.gradle]: Yes, I was able to double check this.

File: app/src/main/java/com/example/life_is_well/SettingsPageMain.kt (hunk 3 above)
Comment (mimif10) [reply app/src/main/java/com/example/life_is_well/SettingsPageMain.kt]: double checked file is current on Mental Wellness branch and was able to push

File: app/build.gradle [hunk 5]
```diff
@@ -34,22 +35,16 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
-
-    buildFeatures {
-        viewBinding true
-    }
-   namespace 'com.example.life_is_well'
 }
```
Comment (jacobburgo) [inline app/build.gradle:42]:
```suggestion

     buildFeatures {
         viewBinding true
     }
    namespace 'com.example.life_is_well'
}
```

File: app/build.gradle [hunk 6]
```diff
@@ -34,22 +35,16 @@ android {
...
 
 dependencies {
-    implementation 'androidx.core:core-ktx:1.8.0'
+    implementation 'androidx.core:core-ktx:1.7.0'
```
Comment (jacobburgo) [inline app/build.gradle]:
```suggestion
    implementation 'androidx.core:core-ktx:1.8.0'
```

File: app/build.gradle [hunk 7]
```diff
@@ -34,22 +35,16 @@ android {
...
     implementation 'androidx.appcompat:appcompat:1.5.0'
     implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-    implementation 'androidx.preference:preference:1.1.+'
-    implementation 'androidx.navigation:navigation-ui-ktx:2.5.2'
+    implementation 'androidx.fragment:fragment-ktx:1.5.2'
```
Comment (jacobburgo) [inline app/build.gradle:50]:
```suggestion
    implementation 'androidx.preference:preference:1.1.+'
    implementation 'androidx.fragment:fragment-ktx:1.5.2'
```

File: app/build.gradle [hunk 8]
```diff
@@ -34,22 +35,16 @@ android {
...
     implementation 'androidx.preference:preference-ktx:1.2.0'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-}
+}
```
Comment (jacobburgo) [inline app/build.gradle]:
```suggestion
}

```

File: app/build.gradle [hunk 9]
```diff
@@ -14,6 +14,7 @@ android {
         versionCode 1
         versionName "1.0"
 
+
```
Comment (jacobburgo) [inline app/build.gradle]:
```suggestion
```

File: app/src/main/AndroidManifest.xml [hunk 10]
```diff
@@ -13,27 +13,21 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.Life_is_well"
         tools:targetApi="31">
-        <activity
-            android:name=".BMICalc"
-            android:exported="false"
-            android:theme="@style/Theme.Life_is_well_home"/>
         <activity
```
Comment (jacobburgo) [inline app/src/main/AndroidManifest.xml:16]:
```suggestion
        <activity
             android:name=".BMICalc"
             android:exported="false"
             android:theme="@style/Theme.Life_is_well_home"/>
         <activity
             android:name=".HealthMacroTracker"
             android:exported="false"
             android:theme="@style/Theme.Life_is_well_home" />
        <activity
```

File: app/src/main/res/layout/activity_main.xml [hunk 11]
```diff
@@ -20,10 +22,10 @@
...
     <FrameLayout
-        android:id="@+id/fragment_container"
+        android:id="@+id/toDo_container"
         android:layout_width="match_parent"
         android:layout_height="0dp"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toTopOf="@id/home_bottom_nav"/>
 
-</androidx.constraintlayout.widget.ConstraintLayout>
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (jacobburgo) [inline app/src/main/res/layout/activity_main.xml]:
```suggestion
</androidx.constraintlayout.widget.ConstraintLayout>

```

File: app/src/main/res/values/colors.xml [hunk 12]
```diff
@@ -20,9 +21,14 @@
...
     <color name="colorBackgroundHome">#1D5284</color>
     <color name="colorDividerHome">#D6D6D6</color>
     <color name="colorLightGrey">#b7b7a4</color>
 
-</resources>
+    <color name="colorOffWhite">#EEEAEA</color>
+
+
+
+</resources>
```
Comment (jacobburgo) [inline app/src/main/res/values/colors.xml]:
```suggestion
</resources>

```

File: app/src/main/res/values/strings.xml [hunk 13]
```diff
@@ -27,38 +43,22 @@
...
-    <string name="attachment_summary_on">Automatically download attachments for incoming emails
-     </string>
-    <string name="attachment_summary_off">Only download attachments when manually requested</string>
+    <string name="nameOfUser">NameOfUser</string>
+    <string name="profile_photo_of_user">Profile Photo of User</string>
+    <string name="user_details">User Details</string>
+    <string name="user_stats">User Stats</string>
+    <string name="bio">Bio</string>
+    <string name="trophy_case">Trophy Case</string>
 </resources>
```
Comment (jacobburgo) [inline app/src/main/res/values/strings.xml]:
```suggestion
</resources>

```

**Review by jacobburgo — CHANGES_REQUESTED** [review]
Some trailing file whitespace and making sure build.gradle is still consistent across everyone. Gonna be something that we won't have to do after we all get merged and pulled and are working from the same starting point.

**Review by jacobburgo — APPROVED** [review]
Good to go :shipit:

**Pull request merged**